presto-main/src/main/java/com/facebook/presto/type/DecimalOperators.java

```diff
@@ -616,4 +619,28 @@ public static long hashCode(@SqlType("decimal(p, s)") Slice value)
             return XxHash64.hash(value);
         }
     }
+
+    @ScalarOperator(INDETERMINATE)
+    public static final class Indeterminate
+    {
+        @LiteralParameters({"p", "s"})
+        @SqlType(StandardTypes.BOOLEAN)
+        public static boolean indeterminate(@SqlType("decimal(p, s)") long value, @IsNull boolean isNull)
+        {
+            if (isNull) {
+                return true;
+            }
+            return false;
+        }
+
+        @LiteralParameters({"p", "s"})
+        @SqlType(StandardTypes.BOOLEAN)
+        public static boolean indeterminate(@SqlType("decimal(p, s)") Slice value, @IsNull boolean isNull)
```

Comment:
I think `@IsNull` is necessary here because I have assumed that `INDETERMINATE` operators always take two arguments. If some `INDETERMINATE` implementations have the number of arguments other than 2, we need an extra step to check the parameters of the `MethodHandle` before invoking the `INDETERMINATE` `MethodHandle` in `ArrayIndeterminate`, `MapIndeterminate` and `RowIndeterminate`. @martint , what do you think about it?